assets/templates/dataset-filter/filter-overview.tmpl

```diff
@@ -35,9 +35,9 @@
                                         <div class="font-size--16 height-sm--3 height-md--10 nowrap-sm vertical-align-middle margin-left-sm--4 list--ellipses-sm overflow--hidden">
```

Comment:
Looks like there are some `font-size--16` left in here. One on line 35, another on 33, may be worth doing a quick search in your code editor and checking to see if any of these files you've changed contain any other instances of that class

Reply:
There was one more in the age.tmpl which I have updated as well.